docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -903,6 +905,15 @@ sources described above to create conditional samplers for first name
 and last name based on soundex.  Perhaps measure of success is to look
 at entropy of character n-gram distribution.
 
+To simulate naming after a parent or family member, have 5% of children 
+born in the simulation be named after a relative that matches their 
+sex. To do this: 
+
+#. If there is a tracked parent of the same sex, assign the same first name 
+#. If there is a relative of the same sex, assign the same first name. Note that this might be a relative of the reference person (i.e., a male child has a tracked mother, but that tracked mother has a tracked father) 
+#. If none of these are available, assign a random name 
```

Comment:
Could you define "tracked parent" here?

Reply:
It might be helpful to separate two concepts:

We have a "relation to household head" attribute for every simulant, and in specific cases (such as ["reference person" and "biological child"] pairs) we can derive information about the relation between two sims, but these are sort of implied relationships -- and they get murky, for example, if we have ["reference person", "parent", "parent"] or other such collections of people we might assume to be married, children of, etc.

We also "track" mothers of children born in the sim. this means that the child sim has an attribute called "parent_id", and this attribute links directly to the unique identifier associated with the mother. I 

I think you guys are considering creating more of these direct links for simulants on initialization, and if you do that, it will be much more straightforward to establish who a "tracked parent" is. however, in that case, i think we should hold off on implementing this until you've decided how to implement "tracked parent"

in the case that you _dont_ end up creating maps from children to parents on initialization, then we'll need a more specific set of rules for who parents are in order to implement this. for example:
- two people who live in the same household and have ["reference person", "biological child"], ["reference person", "adopted child"],
- two people who live in the same household and have ["reference person", "parent"] (if there are multiple parents of the same sex, we would want a way to choose which parent
- grandparent/parent
- grandchild/child
- etc

Reply:
@SylLutze correct me if I'm wrong, but I read "tracked parent" here to refer exclusively to the existing mother ID and a (possibly implemented in future) father ID. So for now, the "tracked parent" line in this logic would only ever happen when a female child is born during the simulation.

Reply:
Yes, I wrote this meaning it to only apply to new people born in the simulation. I intended "tracked parent" as the mother we are tracking right now

Reply:
ohh i see. that makes sense.

could you define for case #2 how the relative is determined and chosen? this is (unfortunately) i think rather complicated. if the mother has a male child, and the household structure is:

- reference person
- sister
- brother
- brother
- parent-in-law
- grandparent

^ then if she is the sister, we need to know how to choose which male relative's name to use
but in the case she is, say, "parent", we would need a different set of rules for which male relative's name to use